src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,377 @@
+align with our ideals.
+
+We currently work directly with Forge on moderation-related issues, and we're looking to work with them on some 
+technical problems. We also contribute relevant fixes to them, for cases where we've forked their projects -- such 
+as with [QuiltFlower](https://github.com/quiltMC/quiltflower) and 
+[ForgeFlower](https://github.com/MinecraftForge/ForgeFlower).
+
+# Questions and Concerns
+
+Below, you'll find a list of common questions and concerns. Because some answers are quite long, they've been 
+collapsed -- please click or tap on them to read our response. Additionally, it's entirely possible that we've 
+forgotten to cover something here -- we'd love to hear from you if you think of anything that needs adding!
+
+{% admonition %}
+{% details %}
+{% summary %}What does Quilt provide over other mod-loaders?{% endsummary %}
+
+At the time of writing, Quilt is in beta -- that means that it's still in a testing phase, so not everything we'd like to
+do has been done quite yet. It also means that many of the things we've done to the project itself are 
+developer-oriented (although this can still result in better mods for everyone).
+
+Right now, the following things are already true:
+
+* Quilt takes a community-focused approach to just about everything, making plenty of room for people to voice their
+  opinions and weigh in on important development decisions before they happen, and working with the community to meet
+  its member's needs in community spaces
+* QuiltFlower, Quilt's decompiler project, has been vastly improved compared to FernFlower and the other FernFlower 
+  forks used in the modding sphere, producing far more readable and more accurate decompiled code for modders to work
+  with
+* Quilt Mappings, Quilt's mappings project, does not have a clean-room policy, which means that
+  developers are welcome to use whatever mappings they'd prefer,
+  and they'll still be able to get mod development support in Quilt community spaces
+* Quilt's Standard Libraries (QSL) already provide a larger API surface than Fabric's API in some instances, such as
+  the QSL dynamic recipe module and the resource loader improvements
+
+The following improvements are also planned for a full release:
+
+* Clearer errors when required mods are missing or mods are incompatible with each other -- user-friendly errors are
+  a large focus of some of our work, and we're hoping to get some dedicated people working on that soon
+* Automatic detection of QSL modules used by mods at compile-time, and automatic downloading of those modules at
+  runtime, eliminating the requirement for users to download the entirety of QSL for Quilt mods using it to
+  function (but this will be able to be reconfigured and disabled by users, if they'd prefer)
+* An inclusive implementation for loader plugins, which will allow developers to modify how Quilt loads mods, and to
+  load them from other sources, if they'd like to
```

Comment:
Patchwork is a bit much, but perhaps we could name-drop something like Sponge here:
"This would allow projects like Sponge to integrate their plugins natively with Quilt Loader"